puppet/zulip/manifests/postfix_localmail.pp

```diff
@@ -45,9 +45,8 @@
     mode    => '0644',
     owner   => root,
     group   => root,
-    source  => 'puppet:///modules/zulip/postfix/virtual',
+    content => template('zulip/postfix/virtual.erb'),
     require => Package[postfix],
-    notify  => Service['postfix'],
```

Comment:
Why are we removing the `notify` here?  I thought that's required to reload postfix configuration after this changes?

Reply:
Changes to `virtual` files don't require a restart.  I'll split this into its own commit.

Reply:
http://www.postfix.org/DATABASE_README.html#detect implies it _may_ be necessary in some cases, so I'll drop this hunk.